system/user/cache/default_site/fieldtype/channel-field-list

```diff
@@ -1 +1 @@
-a:3:{s:4:"time";i:1642002398;s:3:"ttl";i:60;s:4:"data";a:3:{s:5:"About";a:1:{s:3:"2_3";s:9:"Page Copy";}s:4:"Blog";a:1:{s:3:"1_1";s:14:"Blog Body Copy";}s:7:"Contact";a:1:{s:3:"3_3";s:9:"Page Copy";}}}
```

Comment:
Wonder if we should gitignore the cache folder. These files don't really serve much purpose in Version Control as they're different for each session